shell/platform/fuchsia/flutter/accessibility_bridge.cc

```diff
@@ -282,22 +282,12 @@ void AccessibilityBridge::UpdateScreenRects(
   const auto& current_transform = parent_transform * node.transform;
 
   const auto& rect = node.rect;
-  SkScalar quad[] = {
-      rect.left(),  rect.top(),     //
-      rect.right(), rect.top(),     //
-      rect.right(), rect.bottom(),  //
-      rect.left(),  rect.bottom(),  //
+  SkV4 dst[2] = {
+      current_transform.map(rect.left(), rect.top(), 0, 1),
+      current_transform.map(rect.right(), rect.bottom(), 0, 1),
   };
-  SkScalar dst[4 * 4];
-  current_transform.map2(quad, 4, dst);
-  node.screen_rect.setLTRB(dst[0], dst[1], dst[8], dst[9]);
+  node.screen_rect.setLTRB(dst[0].x, dst[0].y, dst[1].x, dst[1].y);
   node.screen_rect.sort();
-  std::vector<SkVector4> points = {
```

Comment:
huh, this was just unused?

Reply:
Yup, it was sort of redundant with the previous chunk of code, and then the result was never used. Also, the previous code transformed all four corners, but then only used the results from two of them.